src/main/java/com/faforever/api/clan/ClansController.java

```diff
@@ -37,11 +35,12 @@
 
   @ApiOperation("Grab data about yourself and the clan")
   @ApiResponses(value = {
-      @ApiResponse(code = 200, message = "Success with JSON { player: {id: ?, login: ?}, clan: { id: ?, name: ?, tag: ?}}"),
-      @ApiResponse(code = 400, message = "Bad Request")})
-  @RequestMapping(path = "/me", method = RequestMethod.GET, produces = APPLICATION_JSON_UTF8_VALUE)
-  public MeResult me(Authentication authentication) {
-    Player player = playerService.getPlayer(authentication);
+    @ApiResponse(code = 200, message = "Success with JSON { player: {id: ?, login: ?}, clan: { id: ?, name: ?, tag: ?}}"),
+    @ApiResponse(code = 400, message = "Bad Request")})
+  @RequestMapping(path = "/me", method = RequestMethod.GET, produces = APPLICATION_JSON_VALUE)
```

Comment:
Maybe we can start using `@{METHOD}Mapping` annotations i.e. `@GetMapping`

Reply:
Yes we can.